homeassistant/components/hdmi_cec.py

```diff
@@ -328,7 +328,6 @@ def __init__(self, hass: HomeAssistant, device, logical) -> None:
         self._state = STATE_UNKNOWN
         self._logical_address = logical
         self.entity_id = "%s.%d" % (DOMAIN, self._logical_address)
-        device.set_update_callback(self._update)
```

Comment:
This should stay or else it will not update on a callback ?

Reply:
Found this unnecessary, thnx!